packages/cli/src/util/telemetry/index.ts

```diff
@@ -90,6 +91,27 @@ export class TelemetryClient {
     });
   }
 
+  protected trackCPUs() {
```

Comment:
Is this a styling decision that we're forcing child classes to implement these? Wondering if there's value in this hierarchy vs just putting these functions directly on `RootTelementClient`, with no ancestor implementation.

Reply:
The child class doesn't have access to the `track` method, which is intentional.